App Review Backend

johnsonwe opened the pull request:
https://github.com/BCStudentSoftwareDevTeam/Ullman_Galleries/issues/6

johnsonwe committed:
Created template for Application Review view

johnsonwe committed:
Merge branch 'development' of https://github.com/BCStudentSoftwareDevTeam/Ullman_Galleries into applicationreviewbackend

johnsonwe committed:
backend functionality for application review

johnsonwe committed:
Merge branch 'development' of https://github.com/BCStudentSoftwareDevTeam/Ullman_Galleries into applicationreviewbackend

Jessonsotoventura commented on app/templates/views/public/application_review.html:
```diff
@@ -3,23 +3,23 @@
 <h2 style="text-align:center;">Gallery</h2>
```
The gallery object is part of the form table, so we can use that to get the current gallery title.

Jessonsotoventura commented on app/templates/views/public/application_review.html:
```diff
@@ -3,23 +3,23 @@
 <div class ="row">
     <label for="name">Name:</label>
-        <p id="name">John Doe</p>
+        <p id="name">{{user.name}}</p>
```
I changed user to form, since we really are not using data stored in the user table, but data stored in the form table

Jessonsotoventura commented on app/controllers/public/application_review.py:
```diff
@@ -2,9 +2,22 @@
+
+@public.route('/application/review/<fid>', methods=["GET","POST"])
+def review(fid):
+    user = Forms.get(fid==fid)
```
For simple common queries, please use the queries in the queries folder.

Jessonsotoventura commented on app/controllers/public/application_review.py:
```diff
@@ -2,9 +2,22 @@
+@public.route('/application/review/<fid>', methods=["GET","POST"])
+def review(fid):
+    user = Forms.get(fid==fid)
+    return render_template('views/public/application_review.html', fid=fid, user = user)
```
It does not look like fid is used in the html, so it should be removed

Jessonsotoventura reviewed (approved):
Nice work Evan! 

I made a few comments about some changes, please have a look at those.

Commit:
Minor changes

- Added get to FormQueries
- Set typing in url
- Pulled gallery title form the db

Commit:
Merge branch 'development' into applicationreviewbackend
